contrib/endpoints/src/api_manager/service_control/info.h

```diff
@@ -38,6 +38,9 @@ namespace service_control {
 
 // Basic information about the API call (operation).
 struct OperationInfo {
+  // The service name
+  ::google::protobuf::StringPiece service_name;
```

Comment:
This will not change.  We don't want to pass in for each request.  We can store it in Mixer class, pass it into in Mixer::Create()